examples/04_fuzzy_joining.py

```diff
@@ -156,44 +171,31 @@
 # Let's do some more inspection of the merging done.
 
 ###############################################################################
-# Let's print the four worst matches, which will give
+# Let's print the worst matches, which will give
 # us an overview of the situation:
 
-df1.sort_values("matching_score").head(4)
+augmented_df.sort_values("skrub_Joiner_rescaled_distance").tail(10)
 
 ###############################################################################
 # We see that some matches were unsuccesful
 # (e.g "Palestinian Territories*" and "Palau"),
 # because there is simply no match in the two tables.
```

Comment:
Actually, re-reading the example, I think this part should go before applying the threshold parameter (and to motivate it). It must have moved down as we added new things.